demos/object_detection_demo/README.md

```diff
@@ -0,0 +1,140 @@
+# Object Detection SSD C++ Demo, Async API Performance Showcase
```

Comment:
Please update readme. We shouldn't call async approach as new. It can be good to rewrite it with @eizamaliev.

Reply:
It was there before. What's your suggestion exactly, completely remove that part ?

Reply:
We could rewrite it

Reply:
If current readme corresponds to this demo, please let's keep it as is. We will update readme with better information together with @eizamaliev in separate PR to avoid delaying of merge of this PR.

Reply:
It doesn't correspond to this demo. It focuses on SSD.

Reply:
Ok